guide/private-key-management/single-user-schemes.md

```diff
@@ -0,0 +1,326 @@
+---
+layout: guide
+title: Personal
+nav_order: 2
+parent: Private key management
+permalink: /guide/private-key-management/single-user-schemes/
+main_classes: -no-top-padding
+---
+
+{% include picture.html
+   image = "/assets/images/guide/private-key-management/personal.jpg"
+   retina = "/assets/images/guide/private-key-management/personal@2x.jpg"
+   mobile = "/assets/images/guide/private-key-management/personal-mobile.jpg"
+   mobileRetina = "/assets/images/guide/private-key-management/personal-mobile@2x.jpg"
+   alt-text = "Personal schemes header illustration"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Personal schemes
+
+Now let's dive in and look at the various schemes that might be suitable for a personal wallet-application with a single-user to understand the differences at a high level.
+
+{% include image.html
+   image = "/assets/images/guide/private-key-management/schemes/personal-schemes.jpg"
+   retina = "/assets/images/guide/private-key-management/schemes/personal-schemes@2x.jpg"
+   alt-text = ""
+   caption = "Cloud backup, manual backup , signing-device, threshold-signatures and multi-key."
+   width = 2012
+   height = 400
+%}
+
+- [Automatic cloud backup]({{ '/guide/private-key-management/single-user-schemes/#automatic-cloud-backup' | relative_url }})
+- [Manual backup / Recovery phrase]({{ '/guide/private-key-management/single-user-schemes/#recovery-phrase' | relative_url }})
+- [External signing device]({{ '/guide/private-key-management/single-user-schemes/#external-signing-device' | relative_url }})
+- [Threshold signatures / Key-sharing]({{ '/guide/private-key-management/single-user-schemes/#threshold-signatures--key-sharing' | relative_url }})
+- [Multi-key]({{ '/guide/private-key-management/single-user-schemes/#multi-key' | relative_url }})
+
+***
+
+## Automatic cloud backup
+
+By using an automatic cloud backup for the private key the user is NOT required to manually backup anything when setting up a new wallet. This is sometimes also called *seedless*, as the user is never exposed to the traditional *seed*, or recovery-phrase that is used to initiate a wallet. 
+
+The target audience with this scheme would be new or novice bitcoin users to whom being exposed to backing up a recovery-phrase as part of onboarding can be baffling and backfire. It could also be a starting point for a progressive security model that lets customers upgrade when the start accumulating more funds. 
```

Comment:
```suggestion
The target audience with this scheme would be new or novice bitcoin users to whom being exposed to backing up a recovery-phrase as part of onboarding can be baffling and backfire. It could also be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds. 
```